### PR TITLE
Neelima-> Added two scenarios in Marketing module and created required methods, locators. 

### PR DESCRIPTION
**Pull request by neelimaboyina4018**
Added two scenarios in Marketing module and created required methods, locators.

### CODE REVIEW

File: src/test/java/pageobjects/SalesforceObjects/CampaignObject.java [hunk 1]
```diff
@@ -150,6 +201,21 @@ public static By clickShowActionsInLeads(String name){
     public static By deleteShowActionInLeads(String name){
         return By.xpath("//table[@role=\"grid\"]//tbody//tr//a[@title='"+name+"']//ancestor::th//following-sibling::td[8]//span[text()='Show more actions']");
     }
+    public static By LeadField(String leadName){
```
Comment (snaphray) [inline src/test/java/pageobjects/SalesforceObjects/CampaignObject.java]: LeadField should start from small letter

**Review by snaphray — CHANGES_REQUESTED** [review]
Please resolve review comment

**Comment by neelimaboyina4018** [comment]
> Please resolve review comment

Updated as per the review comments.

**Review by snaphray — APPROVED** [review]
Code looks fine

**Pull request merged**